src/Workspaces/CSharp/Portable/Classification/Worker_DocumentationComments.cs

```diff
@@ -3,9 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
+using static Humanizer.In;
```

Comment:
😕 Do you know what brought this one in?

Reply:
I have `Show completions from unimported namespaces`, so this was probably my mistype, that caused such `using` to be added